src/Chapter1.hs

```diff
@@ -284,7 +284,7 @@ ghci> 1 + 2
 
 > It may look scary to you, but we will cover all this 'Num' and "=>" later. For
   now, you can think of this as a polymorphic function — in this case, the
-  operator, that can work with any numeric types, including 'Int's, 'Double's,
```

Comment:
This is the haddock syntax to have a link on `Int` documentation. As these are Haskell comment format of info we decided to go with it 🙂

Reply:
Makes sense. Didn't know that. Not much experience with Haddock. Better look into that. : )